.travis.yml

```diff
@@ -36,7 +36,8 @@ env:
   - REQUIREMENTS=devel REXTRAS=docs
 
 python:
-  - "2.7"
+  - "3.3"
```

Comment:
@emanueldima Yes, There are some incompatibilities between those versions, and if we enable one day users to add plugins or customize their B2Share, they might use libraries which work only for older versions of python.
Also the overhead is not so big, most of the issues we had with Travis before where because it was working in 2.7 and not in 3.* or the contrary. It should happen a lot less now that we support only 3.*, thus testing locally will mean most of the time that it will work on Travis.

If for some reason there is a need to have very fast feedback from Travis, for example when we edit the `.travis.yml` file, it is still possible to comment the python versions while the PR is "in-progress" and uncomment them before merging. We just have to be extremely careful to not forget the uncommenting part and all reviewers for the `.travis.yml` should check it.